daemon/start.go

```diff
@@ -182,13 +182,9 @@ func (daemon *Daemon) containerStart(ctx context.Context, container *container.C
 		return translateContainerdStartErr(container.Path, container.SetExitCode, err)
 	}
 
-	// Passing ctx to ctr.Start caused integration tests
-	// to be stuck in the cleanup phase
-	ctrCtx := context.TODO()
-
 	// TODO(mlaventure): we need to specify checkpoint options here
-	tsk, err := ctr.Start(ctrCtx, checkpointDir,
-		container.StreamConfig.Stdin() != nil || container.Config.Tty,
+	tsk, err := ctr.Start(context.TODO(), //Passing ctx to ctr.Start caused integration tests to be stuck in the cleanup phase
```

Comment:
```suggestion
	tsk, err := ctr.Start(context.TODO(), // Passing ctx to ctr.Start caused integration tests to be stuck in the cleanup phase
```
Comments without a leading space are reserved for machine-parseable directives.

Reply:
Thanks!